pmd-visualforce/src/main/java/net/sourceforge/pmd/lang/vf/DataType.java

```diff
@@ -106,8 +105,9 @@ public static DataType fromString(String value) {
     /**
      * Map to correct instance, returns {@code Unknown} if the value can't be mapped.
      */
-    public static DataType fromBasicType(BasicType value) {
```

Comment:
However, this method could potentially be used by somebody outside of PMD's code base.

In #4081 we deprecated the existing methods but didn't remove them. I think, we should do here the same (at least on the master branch).

Reply:
Good point. I'll add this back.